crates/kitsune_p2p/kitsune_p2p/src/test_util/switchboard/switchboard_evt_handler.rs

```diff
@@ -129,35 +128,34 @@ impl KitsuneP2pEventHandler for SwitchboardEventHandler {
         &mut self,
         space: Arc<KitsuneSpace>,
         to_agent: Arc<KitsuneAgent>,
-        from_agent: Arc<KitsuneAgent>,
         payload: Vec<u8>,
     ) -> KitsuneP2pEventHandlerResult<()> {
         todo!()
     }
 
     fn handle_gossip(
         &mut self,
-        space: Arc<KitsuneSpace>,
-        to_agent: Arc<KitsuneAgent>,
-        ops: Vec<(Arc<KitsuneOpHash>, Vec<u8>)>,
+        _space: Arc<KitsuneSpace>,
+        _ops: Vec<(Arc<KitsuneOpHash>, Vec<u8>)>,
     ) -> KitsuneP2pEventHandlerResult<()> {
-        ok_fut(Ok(self.sb.share(|sb| {
-            let agent = sb
-                .node_for_local_agent_hash_mut(&*to_agent)
-                .unwrap()
-                .local_agent_by_hash_mut(&*to_agent)
-                .unwrap();
-            for (hash, op_data) in ops {
-                let loc8 = hash.get_loc().as_loc8();
-                // TODO: allow setting integration status
-                agent.ops.insert(
-                    loc8,
-                    AgentOpEntry {
-                        is_integrated: true,
-                    },
-                );
-            }
-        })))
+        todo!()
+        // ok_fut(Ok(self.sb.share(|sb| {
+        //     let agent = sb
+        //         .node_for_local_agent_hash_mut(&*to_agent)
+        //         .unwrap()
+        //         .local_agent_by_hash_mut(&*to_agent)
+        //         .unwrap();
+        //     for (hash, op_data) in ops {
+        //         let loc8 = hash.get_loc().as_loc8();
+        //         // TODO: allow setting integration status
+        //         agent.ops.insert(
+        //             loc8,
+        //             AgentOpEntry {
+        //                 is_integrated: true,
+        //             },
+        //         );
+        //     }
+        // })))
```

Comment:
@maackle This is what I was talking to you about. It doesn't make sense to do things like `.node_for_local_agent_hash_mut(&*to_agent)`. Any idea how we can change this?